ocaml/idl/datamodel_cluster_host.ml

```diff
@@ -66,6 +66,17 @@ let disable = call
     ~allowed_roles:_R_POOL_ADMIN
     ()
 
+let forget = call
+  ~name:"forget"
+  ~doc:"Permanently remove a dead host from the cluster. This host must never rejoin the cluster."
+  ~params:
+      [ Ref _cluster_host, "self", "the cluster_host to declare permanently dead and forget"
+      ]
+  ~lifecycle:[Prototyped, rel_lima, ""]
+  ~allowed_roles:_R_LOCAL_ROOT_ONLY
```

Comment:
Is this intentional, to make it only work from the CLI?

Reply:
You are supposed to use Host.destroy, Cluster_host.forget is just an internal API used by XAPI. We need message-forwarding to send this request to a host that has clustering.

Reply:
We could mark it as hidden in records.ml